client/landing/stepper/utils/enhanceFlowWithAuth.ts

```diff
@@ -2,7 +2,10 @@ import type { Flow, StepperStep } from '../declarative-flow/internals/types';
 
 const USER_STEP: StepperStep = {
 	slug: 'user',
-	asyncComponent: () => import( '../declarative-flow/internals/steps-repository/__user' ),
+	asyncComponent: () =>
+		import(
+			/* webpackChunkName: "stepper-user-step" */ '../declarative-flow/internals/steps-repository/__user'
+		),
```

Comment:
It's hopefully an additional reminder, [to the one in README](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/__user/README.md), to prevent people from using it directly.